src/lib/charts/chart.css

```diff
@@ -2,16 +2,26 @@
   &-label {
     display: flex;
     align-items: center;
+    white-space: nowrap;
     @apply lc-text-sm;
     @apply lc-text-black;
-    @apply lc-font-bold;
+
+    &-name {
+      @apply lc-font-bold;
+    }
 
     &-color {
       display: inline-block;
-      width: theme('spacing.8');
-      height: theme('spacing.8');
+      width: 11px;
+      height: 11px;
       margin-right: theme('spacing.12');
-      border-radius: theme('borderRadius.sm');
+      border-radius: 100%;
+    }
+
+    &-value {
+      &::before {
+        content: ': ';
+      }
     }
```

Comment:
Why not just put this in the component? I'm fine with using pseudo-elements, but this seems unnecessary.